cmd/wfx/cmd/config/flags.go

```diff
@@ -64,6 +65,8 @@ const (
 const (
 	preferedStorage   = "sqlite"
 	sqliteDefaultOpts = "file:wfx.db?_fk=1&_journal=WAL"
+
+	DefaultPingIntervalSSE = 30 * time.Second // should be "short enough", i.e. shorter than the default read timeout of most reverse proxies
```

Comment:
... since this time period is shorter than the one for closing connections due to inactivity by the kernel in its default setting.